kinetic/distribution.yaml

```diff
@@ -0,0 +1,14 @@
+%YAML 1.1
+# ROS distribution file
+# see REP 143: http://ros.org/reps/rep-0143.html
+---
+release_platforms:
+  fedora:
+  - '22'
+  - '23'
```

Comment:
Yeah, it's unfortunate, but it's hard to see how we can cover all the releases with their cycle going twice as fast as ours. If it's only going to be a gap of 1 fedora distro then the users should always have a supported one. Though there may be lag as we spin up, like Wily.